framework_tvb/tvb/interfaces/web/templates/jinja2/visualizers/commons/scripts/regionSelector.js

```diff
@@ -254,7 +254,7 @@ RegionSelectComponent.prototype.selectedIndices = function(arg){
     }else{
         this._set_indices(arg);
         this._dropDown.val("[]");
-        this.$dom.trigger("selectionChange", [this._selectedValues.slice()]);
```

Comment:
This removal is from another task, right?

Reply:
Yes